provider/equinix/environ.go

```diff
@@ -177,11 +249,173 @@ func newConfig(cfg, old *config.Config) (*environConfig, error) {
 }
 
 func (e *environ) StartInstance(ctx context.ProviderCallContext, args environs.StartInstanceParams) (result *environs.StartInstanceResult, resultErr error) {
-	panic(errors.NewNotImplemented(nil, "not implemented"))
+	instanceTypes, err := e.InstanceTypes(ctx, constraints.Value{})
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	spec, err := e.findInstanceSpec(
+		args.InstanceConfig.Controller != nil,
+		args.ImageMetadata,
+		instanceTypes.InstanceTypes,
+		&instances.InstanceConstraint{
+			Region:      e.cloud.Region,
+			Series:      args.InstanceConfig.Series,
+			Arches:      args.Tools.Arches(),
+			Constraints: args.Constraints,
+		},
+	)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	if err := e.finishInstanceConfig(&args, spec); err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	cloudCfg, err := cloudinit.New(args.InstanceConfig.Series)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	cloudCfg.AddScripts(
+		// This is a dummy script injected into packet images that
+		// confuses the init system detection logic used by juju.
+		"rm -f /sbin/initctl",
+	)
+
+	// Install additional dependencies that are present in ubuntu images
+	// but not in the versions built by equinix.
+	//
+	// NOTE(achilleasa): this is a hack and is only meant to be used
+	// temporarily; we must ensure that equinix mirrors the official
+	// ubuntu cloud images.
```

Comment:
If this is not a concern anymore we should remove this comment. @petevg thoughts?

Reply:
It may be some time before we can bundle the official images. 

My request to proxy cloud images on mirrors.packet.net (and the EM counterpart) was closed in favor of future alternatives.  Even if we did this, we would want to have https://github.com/canonical/cloud-init/pull/680 which would be further out.

Reply:
@thebsdbox Do you know more about how/when we would be able to provide our own copy of the official cloud images?

Reply:
@jnsgruk @jameinel what are the thoughts of an ubuntu image that isn't an official cloud image?

Reply:
I think using the official Ubuntu cloud images is definitely preferable, and in any case that we're using a non-standard image it should be labelled clearly as such to any users for sure.